server/models/user.model.js

```diff
@@ -17,6 +17,10 @@ const UserSchema = new mongoose.Schema({
       type: String,
       required: true,
     },
+    isAdmin: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
What keeps a user with knowledge of the API from sending a user data update with {isAdmin:true} and gaining admin?

Reply:
At this time, the only time the property is set, is when the user is created, and as far as I know, there isn't a way to override a set default. There are no api calls to change this property from false to true. It can only be changed manually.

Reply:
Is there an API call to change the user profile data though? Can you set it as readOnly somehow?

Reply:
I probably could, but all user data changes are hard coded like, "userInfo: derp" and there is no mention of the admin property in any of the calls.